test/e2e/plain_provisioner_test.go

```diff
@@ -64,37 +64,33 @@ var _ = Describe("plain provisioner bundle", func() {
 
 		It("should eventually report a successful state", func() {
 			By("eventually reporting an Unpacked phase", func() {
-				Eventually(func() bool {
+				Eventually(func() (string, error) {
 					if err := c.Get(ctx, client.ObjectKeyFromObject(bundle), bundle); err != nil {
-						return false
+						return "", err
 					}
-					return bundle.Status.Phase == rukpakv1alpha1.PhaseUnpacked
-				}).Should(BeTrue())
+					return bundle.Status.Phase, nil
+				}).Should(Equal(rukpakv1alpha1.PhaseUnpacked))
 			})
 
 			By("eventually writing a non-empty image digest to the status", func() {
-				Eventually(func() bool {
+				Eventually(func() (string, error) {
 					if err := c.Get(ctx, client.ObjectKeyFromObject(bundle), bundle); err != nil {
-						return false
+						return "", err
 					}
-					// Note(tflannag): This may make the test fairly brittle over time. Let's re-evaluate
-					// whether we want to test this kind of comparison if there's the potential for moving
-					// tags for Bundle container images.
-					expectedDigest := "quay.io/tflannag/olm-plain-bundle@sha256:2a72e4a7bc6c7598d1ecdb2f082c865a91dda35ab8e4a8e8bc128cf49a2a619b"
-					return bundle.Status.Digest == expectedDigest
-				}).Should(BeTrue())
+					return bundle.Status.Digest, nil
+				}).Should(Not(Equal("")))
```

Comment:
I changed this to just check for a non-empty digest to make this test less brittle (as the note alluded to). IMO this is sufficient for now and aligns with the actual text of the test.

Reply:
Yep this is fine with me.